cli/test/e2e/dev_test.go

```diff
@@ -79,4 +102,105 @@ spec:
 		}
 		require.Exactly(t, expectedOutputRows, replaceAgeAndSinceDeployed(output.Tables[0].Rows))
 	})
+
+	logger.Section("dev deploy git app", func() {
+		out, err := kappCtrl.RunWithOpts([]string{"dev", "-f", "-"}, RunOpts{StdinReader: strings.NewReader(gitAppYaml)})
+		fmt.Printf("\n\n Out: %s \n err: %+v ", out, err)
+	})
+
+	logger.Section("inspect gitApp app resources", func() {
```

Comment:
Using kctrl, we can get the app status and ensure that it is reconciled successfully. I think kapp inspect is bit too much.

Reply:
Here we are using local reconciler, so we cannot get the app status using kctrl as there is no real app resource created in the cluster.

Reply:
Or else shall we use kapp as it is and reduce the number of expected rows only with `Deployment` and `Service` kind objects?
(existing test case for kctrl dev already uses kapp to verify the resources, this test case developed as per the existing
 one)